R/zzz.R

```diff
@@ -0,0 +1,3 @@
+.onUnload <- function (libpath) {
```

Comment:
looks like linting is not set up, I have a feeling that needs to be done by me, I'll do that

Reply:
Good catch, will add that here and I've made a maintenance ticket https://mrc-ide.myjetbrains.com/youtrack/issue/RESIDE-299